txpool/txpool.go

```diff
@@ -589,6 +599,66 @@ func (t *TxPool) decreaseSlots(slots uint64) {
 	atomic.AddUint64(&t.slots, ^(slots - 1))
 }
 
+func (t *TxPool) isSpeedUp(tx *types.Transaction) (*types.Transaction, bool) {
```

Comment:
Please document this method

Reply:
[Documented](https://github.com/0xPolygon/polygon-sdk/pull/220/commits/8e4ce12c2c344c912148fb5a04a2c386774a0836)